lib/_cookieUtils.ts

```diff
@@ -65,6 +65,6 @@ const createUUID = () => {
 };
 
 const sessionID = createUUID;
```

Comment:
`sessionToken`?

Reply:
actually i'll remove it altogether because it's unused and I don't think we need to expose it externally.